<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EpipolarDialog</class>
 <widget class="QDialog" name="EpipolarDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="closePushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadFirstImgPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load First Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadSecondImgPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Second Image</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>F Matrix</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Method</string>
    </property>
   </widget>
   <widget class="QComboBox" name="FmatMethodComboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>select</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7 Point Method</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 Point Method</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RANSAC</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>251</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Epipolar Lines</string>
   </property>
   <widget class="QCheckBox" name="showInFirstCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show in First Image</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showInSecondCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show in Second Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Confidence Level</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Epipolar Ratio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Hessian</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="confidenceDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <double>2.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="epipolarRationSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <double>1.500000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="minHessianSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="minDistDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>66</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <double>5.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Distance</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="showEpiLinesPushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
